encoding/resampling.md

```diff
@@ -1,35 +1,429 @@
+```
+
+##### Spline
+
+Spline is another high-quality resizer.
+
+Spline, like Lanczos,
+can be finetuned by configuring its number of lobes.
+Unlike Lanczos, 
+however,
+Splines with different tap counts are usually split
+into seperate functions,
+with ``(tap_count*2)**2`` appended to their name,
+e.g. Spline36 for 3 taps, Spline64 for 4, etc.
+(This number repesents the total amount of input pixels
+involved in the calculation of any given output pixel.)
+
+Spline36 is a very popular choice for downscaling,
+since it is fairly artifact-free yet decently sharp.
+For upscaling,
+it looks similiar to Lanczos3,
+though arguably slightly less artifacted.
+
+VS example:
+
+```py
+clip = core.resize.Spline36(src, w, h)
+```
+
+Higher tap counts can be used via fmtconv:
+
+```py
+clip = core.fmtc.resample(src, w, h, kernel="spline", taps=6) # equivalent to Spline144
+```
+
+##### Gauss
+
+The Gaussian filter is very special in that its [Fourier transform][^1]
+is another Gaussian
+whose width is antiproportional to the spatial function’s.
+This can be harnessed to remove and amplify high frequencies 
+in a very controllable way.
+Widening the filter,
+for example,
+will confine the output to small frequencies (blurrier),
+whereas squashing it will amplify higher frequencies (more aliasing).
+
+In practice,
+though,
+the Gaussian filter isn’t all too useful for regular resizing.
+However,
+it can be used to accurately
+emulate a Gaussian blur (when used without resizing) in VapourSynth.
+
+For example:
+
+```py
+blurred = core.fmtc.resample(src, kernel="gauss", fh=-1, fv=-1, a1=1)
+```
+
+``fh=-1, fv=-1`` forces the processing when no resizing is performed.
+``a1`` controls the blurring:
+the higher,
+the sharper the image.
+
+
+#### Interpolation filters
+
+It is sometimes distinguished between 
+interpolation filters and non-interpolation filters.
+
+Interpolation filters are those that, 
+when applied “in-place”,
+i.e. at the location of the input samples,
+don’t alter the sample value.
+Therefore,
+they only interpolate “missing” values,
+leaving the input samples untouched.
+
+This is true for filters that evaluate to 0
+at all integer positions except 0
+or whose support is <= 1.
+Examples include:
+(Windowed) Sinc filters,
+such as Lanczos,
+Bicubic with B=0,
+e.g. Hermite and Catmull-Rom,
+and Triangle/Bilinear.
+
+This can be a beneficial property in some cases,
+for example the No-Op case.
+No-Op means that no scaling, shifting or similiar is performed,
+that is, the input is resampled at exactly the same positions.
+In this case,
+an interpolation filter will return the input image untouched,
+whereas other filters will slightly alter it.
+
+Another, more practical,
+case where this property is useful
+is when shifting an image by full pixel widths (integers),
+again because input pixel values aren’t changed
+but just relocated.
+
+#### Two-dimensional resampling
```

Comment:
Idk which heading level would be appropriate for this, but you may want to reconsider after the above comments. I guess h3 makes the most sense.